pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Summary
+
+<!-- Required section. Describe what this PR will do if it is merged. -->
+
+## Details
+
+<!-- Optional section. Give more details if it is sensible to do so. -->
+
+## Tests
+
+<!-- Required section. Run the tests and ensure that they pass. If only specific tests were run for this PR, indicate that here. -->
+
+Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.
```

Comment:
seems useful, but would be nice to have inline so it's a better DX vs TLDR.

i like what cloud-ui is doing, and seems similar to conventional comments. is very clear to any developer what to do as it's written out in the template. what is your opinion on it? https://github.com/hashicorp/cloud-ui/blob/master/.github/pull_request_template.md

Reply:
That’s nice! I don’t like the mountain/pebble/dust thing though. More things I have to translate the relative meanings of- I think it’s more useful to use the exact word instead. We could otherwise simply adopt their PR template, it seems fine. Would that work for you?

Reply:
sure. what about taking their template but adding some inline documentation to conventional comments? e.g. 

```suggestion
Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR. For example:

issue (blocking): 
nitpick (non-blocking):
```